devops/docker/ubuntu-14.04-amd64/Dockerfile

```diff
@@ -2,34 +2,11 @@ FROM ubuntu:trusty
 
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt -y update && apt-get -y install software-properties-common
-RUN apt -y update && apt-get -y install \
-    apt-transport-https \
+RUN apt-get -y update && apt-get -y install \ 
+    gcc \
     git \
     cmake \
     build-essential \
-    curl \
-    libcurl4-openssl-dev \
-    libssl-dev \
-    uuid-dev \
-    ninja-build \
-    python3-jsonschema \
-    wget \
-    gcovr\
-    jq \
-    bc \
     file
 
-WORKDIR /git
-
-# CMake
-RUN git clone https://github.com/Kitware/CMake --recursive -b v3.21.7
```

Comment:
Do we need CMake to build on Ubuntu 14?

Reply:
Yes, but the 'inbox' 2.x one is enough, no need for v 3.

Reply:
Ahh I see, thanks for the clarification  👍